packages/languages/jest.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
I think you don't need `rootDir` or `cacheDirectory`, they come from the `preset`

Reply:
[`4ea1e04` (#46709)](https://github.com/Automattic/wp-calypso/pull/46709/commits/4ea1e0437b0dc126993ef86634c563a553d083a3)